unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridController.cs

```diff
@@ -268,6 +286,65 @@ private async UniTask<int> RequestWearablesAndShowThem(int page, CancellationTok
             return 0;
         }
 
+        private async UniTask<IEnumerable<WearableItem>> FetchCustomWearableItems(CancellationToken cancellationToken)
+        {
+            IReadOnlyList<string> customItems = await customNftCollectionService.GetConfiguredCustomNftItemsAsync(cancellationToken);
+
+            IEnumerable<string> publishedItems = customItems
+               .Where(nftId => nftId.StartsWith("urn", StringComparison.OrdinalIgnoreCase));
+
+            IEnumerable<string> itemsInBuilder = customItems
+               .Where(nftId => !nftId.StartsWith("urn", StringComparison.OrdinalIgnoreCase));
+
+            List<WearableItem> wearables = new ();
+
+            foreach (string nftId in publishedItems)
+            {
+                WearableItem wearable = await wearablesCatalogService.RequestWearableAsync(nftId, cancellationToken);
+
+                if (wearable == null)
+                {
+                    Debug.LogWarning($"Custom wearable item skipped is null: {nftId}");
+                    continue;
+                }
+
+                wearables.Add(wearable);
+            }
+
+            foreach (string nftId in itemsInBuilder)
+            {
+                WearableItem wearable = await wearablesCatalogService.RequestWearableFromBuilderAsync(nftId, cancellationToken);
+
+                if (wearable == null)
+                {
+                    Debug.LogWarning($"Custom wearable item skipped is null: {nftId}");
+                    continue;
+                }
+
+                wearables.Add(wearable);
+            }
+
+            return wearables;
+        }
+
+        private async UniTask<IEnumerable<WearableItem>> FetchCustomWearableCollections(CancellationToken cancellationToken)
```

Comment:
You don't need to create a collection explicitly: just pass a collection you want to fill